From: yangfl <yangfl@users.noreply.github.com>
Date: Fri, 19 Oct 2018 23:21:53 +0800
Subject: Do not download intersphinx mappings

Forwarded: not-needed
---
 doc/conf.py | 11 +++++------
 1 file changed, 5 insertions(+), 6 deletions(-)

diff --git a/doc/conf.py b/doc/conf.py
index 127fce8..b3e2e90 100644
--- a/doc/conf.py
+++ b/doc/conf.py
@@ -571,13 +571,12 @@ trim_doctests_flags = True
 
 # intersphinx configuration
 intersphinx_mapping = {
-    "python": ("https://docs.python.org/{.major}".format(sys.version_info), None),
-    "numpy": ("https://numpy.org/doc/stable", None),
-    "scipy": ("https://docs.scipy.org/doc/scipy/", None),
+    "python": ("/usr/share/doc/python3/html/", None),
+    "numpy": ("/usr/share/doc/python-numpy/html/", None),
+    "scipy": ("/usr/share/doc/python-scipy-doc/html/", None),
     "narwhals": ("https://narwhals-dev.github.io/narwhals/", None),
-    "matplotlib": ("https://matplotlib.org/", None),
-    "pandas": ("https://pandas.pydata.org/pandas-docs/stable/", None),
-    "joblib": ("https://joblib.readthedocs.io/en/latest/", None),
+    "matplotlib": ("/usr/share/doc/python-matplotlib/html/", None),
+    "pandas": ("/usr/share/doc/python-pandas-doc/html/", None),
     "seaborn": ("https://seaborn.pydata.org/", None),
     "skops": ("https://skops.readthedocs.io/en/stable/", None),
 }
