From: yangfl <yangfl@users.noreply.github.com>
Date: Fri, 19 Oct 2018 23:42:39 +0800
Subject: Disable doctest to prevent Python 2 doctest failure

===================================================================
---
 setup.cfg | 1 -
 1 file changed, 1 deletion(-)

diff --git a/setup.cfg b/setup.cfg
index f086993..cce94be 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -11,7 +11,6 @@ addopts =
     --ignore doc
     --ignore examples
     --ignore maint_tools
-    --doctest-modules
     --disable-pytest-warnings
     -rxXs
 
