From: =?UTF-8?q?Lo=C3=AFc=20Est=C3=A8ve?= <loic.esteve@ymail.com>
Subject: Fix i686 test failure
Forwarded: https://github.com/scikit-learn/scikit-learn/issues/27506
Bug-Debian: https://bugs.debian.org/1070201

diff --git a/sklearn/tree/_export.py b/sklearn/tree/_export.py
index dd3c655173..43909a17ee 100644
--- a/sklearn/tree/_export.py
+++ b/sklearn/tree/_export.py
@@ -266,7 +266,7 @@ class _BaseTreeExporter:
             self.colors["rgb"] = _color_brew(tree.n_classes[0])
             if tree.n_outputs != 1:
                 # Find max and min impurities for multi-output
-                self.colors["bounds"] = (np.min(-tree.impurity), np.max(-tree.impurity))
+                self.colors["bounds"] = (-np.max(tree.impurity), -np.min(tree.impurity))
             elif tree.n_classes[0] == 1 and len(np.unique(tree.value)) != 1:
                 # Find max and min values in leaf nodes for regression
                 self.colors["bounds"] = (np.min(tree.value), np.max(tree.value))
