--- a/sklearn/_build_utils/__init__.py
+++ b/sklearn/_build_utils/__init__.py
@@ -63,14 +63,7 @@ def cythonize_extensions(top_path, confi
     #   to actually build the compiled extensions with OpenMP flags if needed.
     sklearn._OPENMP_SUPPORTED = check_openmp_support()
 
-    n_jobs = 1
-    with contextlib.suppress(ImportError):
-        import joblib
-        if LooseVersion(joblib.__version__) > LooseVersion("0.13.0"):
-            # earlier joblib versions don't account for CPU affinity
-            # constraints, and may over-estimate the number of available
-            # CPU particularly in CI (cf loky#114)
-            n_jobs = joblib.cpu_count()
+    n_jobs = int(os.environ.get('JOBS', 1))
 
     config.ext_modules = cythonize(
         config.ext_modules,
